### PR TITLE
Build temp static /data/contact-us form

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

- Build a static form based on the [copydoc](https://docs.google.com/document/d/1MIfxsM1cFetY9hYttT6eZO98OcXmirLqwdA_X3-1bIA/edit). The form is temporary and will be replaced by the /data modal form, [see issue](https://warthogs.atlassian.net/browse/WD-5960)

## QA

- Visit /data/contact-us and fill out the form, submit
- Check payload contains the data and form id matches
- Check form is delivered to marketo

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-4913

### CODE REVIEW

File: templates/data/contact-us.html [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+            <li class="p-list__item">
+              <label for="companyName">Company name:</label>
+              <input required id="companyName" name="companyName" maxlength="255" type="text"/>
+            </li>
+            <li class="p-list__item">
+              <label for="jobTitle">Job title:</label>
+              <input required id="jobTitle" name="jobTitle" maxlength="255" type="text"/>
+            </li>
+            <li class="p-list__item">
+              <label for="email">Work address:</label>
```
Comment (mtruj013) [inline templates/data/contact-us.html:48]:
```suggestion
              <label for="email">Work email:</label>
```

File: templates/data/contact-us.html [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+          <legend class="u-off-screen">Privacy policy</legend>
+          <ul class="p-list">
+            <li class="p-list__item">
+              <label class="p-checkbox">
+                <input class="p-checkbox__input" value="yes" aria-labelledby="canonicalUpdatesOptIn" name="canonicalUpdatesOptIn" type="checkbox">
+                <span class="p-checkbox__label" id="canonicalUpdatesOptIn">I'd like to get information about Canonical's products and services.</span>
+              </label>
+            </li>
+            <li class="p-list__item">
+              In submitting this form, I confirm that I have read and agree to <a href="/legal/data-privacy/contact">Canonical's Privacy Notice</a> and <a href="/legal/data-privacy">Privacy Policy</a>.
```
Comment (mtruj013) [inline templates/data/contact-us.html:69]: Copy for this is a bit different on the doc

File: templates/data/contact-us.html [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+  </div>
+  <div class="row">
+    <div class="col-8">
+      <form action="/marketo/submit" method="post" id="mktoForm_5037">
+        <fieldset class="u-no-margin--bottom">
+          <legend class="u-off-screen">Tell us about your data related use case:</legend>
+          <h3>Tell us about your data related use case:</h3>
+          <ul class="p-list">
+            <li class="p-list__item">
+              <label for="Comments_from_lead__c">What would you like to talk to us about?</label>
```
Comment (mtruj013) [inline templates/data/contact-us.html:22]: This copy is missing on doc

File: templates/data/contact-us.html [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+          <input type="hidden" aria-hidden="true" aria-label="hidden field" name="utm_content" id="utm_content" value="" />
+          <input type="hidden" aria-hidden="true" aria-label="hidden field" name="utm_term" id="utm_term" value="" />
+          <input type="hidden" aria-hidden="true" aria-label="hidden field" name="GCLID__c" id="GCLID__c" value="" />
+          <input type="hidden" aria-hidden="true" aria-label="hidden field" name="Facebook_Click_ID__c" id="Facebook_Click_ID__c" value="" />
+          <input type="hidden" aria-hidden="true" aria-label="hidden field" id="preferredLanguage" name="preferredLanguage" maxlength="255" value="" />
+        </fieldset>
+      </form>
+    </div>
+
+    <div class="col-4">
```
Comment (mtruj013) [inline templates/data/contact-us.html:103]: Not sure if this is relevant (maybe the pattern for single page forms is to always have this), but this section isn't referenced on the doc

File: templates/data/contact-us.html [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+  <div class="row">
+    <div class="col-8">
+      <form action="/marketo/submit" method="post" id="mktoForm_5037">
+        <fieldset class="u-no-margin--bottom">
+          <legend class="u-off-screen">Tell us about your data related use case:</legend>
+          <h3>Tell us about your data related use case:</h3>
+          <ul class="p-list">
+            <li class="p-list__item">
+              <label for="Comments_from_lead__c">What would you like to talk to us about?</label>
+              <textarea required id="Comments_from_lead__c" name="Comments_from_lead__c" rows="4" maxlength="2000"></textarea>
```
Comment (mtruj013) [inline templates/data/contact-us.html:23]: Not required on the doc

File: templates/data/contact-us.html [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+              <label for="jobTitle">Job title:</label>
+              <input required id="jobTitle" name="jobTitle" maxlength="255" type="text"/>
+            </li>
+            <li class="p-list__item">
+              <label for="email">Work address:</label>
+              <input required id="email" name="email" maxlength="255" type="email" pattern="^[^ ]+@[^ ]+\.[a-z]{2,26}$" />
+            </li>
+            <li class="p-list__item">
+              <label for="phone">Phone number:</label>
+              <input required id="phone" name="phone" maxlength="255" type="tel"/>
```
Comment (mtruj013) [inline templates/data/contact-us.html:53]: Not required

**Review by mtruj013 — APPROVED** [review]
Thanks @petesfrench ! LGTM, just a few suggestions in the code

File: templates/data/contact-us.html (hunk 4 above)
Comment (petesfrench) [reply templates/data/contact-us.html:103]: It appears on most other static forms. Don't think it matters too much as this form is just temporary to capture some early leads, but can remove regardless

**Comment by petesfrench** [comment]
@mtruj013 can you take another look please

**Comment by petesfrench** [comment]
Closing as this work is no longer required. The data pdm will wait for the completion of the new `/data` page in October and we will build a modal form there.

**Pull request closed without merging**